server/controllers/BookController.js

```diff
@@ -75,6 +77,44 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+ * Add a new book
+ * @name addBook
+ * @param {Object} request
+ * @param {Object} response
+ * @returns {JSON} object
+ */
+  static async getBorrowingStats(request, response) {
+    try {
+      const { userId } = request.params;
+      const { id, UserRoles: [UserRole] } = request.userData.dataValues;
+      let filter = { userId: id };
+      const { roleName } = UserRole.Role.dataValues;
+      if (roleName === 'admin' || roleName === 'superadmin') {
+        filter = {};
+        if (userId) filter = { userId };
+      }
+      const { limit, page } = request.query;
+      const { count } = await LendingHistory.findAndCountAll({ where: filter });
+      if (!count) return responseMessage(response, 400, { status: 'failure', message: 'no book borrowed yet' });
```

Comment:
This is not a bad request. Kindly use an appropriate status code

Reply:
Changed